src/components/wrappers/ProcessWrapper.tsx

```diff
@@ -113,21 +122,30 @@ export const ProcessWrapper = () => {
 
   if (taskType === ProcessTaskType.Feedback) {
     return (
-      <PresentationComponent type={taskType}>
+      <PresentationComponent type={realTaskType}>
         <Feedback />
       </PresentationComponent>
     );
   }
 
   if (taskType === ProcessTaskType.Archived) {
     return (
-      <PresentationComponent type={taskType}>
+      <PresentationComponent type={realTaskType}>
+        <ReceiptContainer />
+      </PresentationComponent>
+    );
+  }
+
+  if (taskType === ProcessTaskType.Data && customReceiptDataModelNotFound) {
+    window.logWarn('You specified a custom receipt, but the data model is missing. Falling back to default receipt.');
```

Comment:
You should use `window.logWarnOnce` since this is called inside of a render function. It will potentially log many times, and there is a risk of causing an infinite render loop if the log somehow causes a rerender. It causes state in the dev-tools to update after all. It is debounced, so it should not cause a freeze, but still

Reply:
Aha! Thanks! I'll fix this right away!